### ISSUE TITLE
In other games

### ISSUE THREAD

**Issue by TiringaMente** [McDaived/AIMi#25]
In other games, the detection does not work, it captures the desktop and not the game

**Comment by peer-pri** [comment]
> In other games, the detection does not work, it captures the desktop and not the game


No. If you do everything correctly and start it you can see that it works in games perfectly. What game did you try?

**Comment by TiringaMente** [comment]
pubg, black squad
